bitnami/pytorch/templates/statefulset.yml

```diff
@@ -0,0 +1,163 @@
+{{- if .Values.pytorch.distributed.enabled }}
+apiVersion: apps/v1beta2
+kind: StatefulSet
```

Comment:
Why `workers` use a statefulset and `master` user a deployment?

Reply:
I'm not sure with this one, but I don't see the advantages of using a StatefulSet over a Deployment.

Reply:
@tompizmor what do you think about this?

Reply:
I won't use a stateful set for the master. We don't need a persistent hostname or persistence volume name